src/extension/background-script/actions/nostr/signEventOrPrompt.ts

```diff
@@ -47,6 +44,8 @@ const signEventOrPrompt = async (message: MessageSignEvent) => {
       }
     }
 
+    event.pubkey = nostr.getPublicKey();
```

Comment:
Not sure about this change. Aren't the clients supposed to fill this event object? 

I think we should only override those properties if they are not set: 

https://github.com/fiatjaf/nos2x/blob/master/extension/background.js#L170-L171

Reply:
Signing clients are supposed to fill those fields according to the spec here: 

https://github.com/nostr-protocol/nips/blob/master/07.md

> // takes an event object, adds `id`, `pubkey` and `sig` and returns it

I'd say we just fill them if they are empty :+1:

Reply:
Ok, just commited a new version with the checks back.